Documentation/OSMPrivatePreview/ReadMe.md

```diff
@@ -30,6 +32,25 @@ osm metrics enable --namespace "test1, test2"
          * Example: `kubectl apply -f container-azm-ms-agentconfig.yaml`
 4. The configuration change can take upto 15 mins to finish before taking effect, and all omsagent pods in the cluster will restart. The restart is a rolling restart for all omsagent pods, not all restart at the same time.
 
+### Azure Arc for Kuberentes
+This section assumes that you already have your kubernetes distribution connected via Azure Arc. If not learn more [here.](https://docs.microsoft.com/en-us/azure/azure-arc/kubernetes/quickstart-connect-cluster)
+
+1. Install Arc enabled Open Service mesh on your Arc cluster. Learn more [here](https://docs.microsoft.com/en-us/azure/azure-arc/kubernetes/tutorial-arc-enabled-osm#navigating-the-osm-dashboard)
```

Comment:
Is Arc enabled open service mesh different from the AKS service mesh?

Reply:
And the link doesnt seem to work.

Reply:
Yes the link will go live soon; the temp link is this:  https://review.docs.microsoft.com/en-us/azure/azure-arc/kubernetes/tutorial-arc-enabled-osm?branch=pr-en-us-158531#navigating-the-osm-dashboard.

The reason I kept it different due to minor installation differences.

Reply:
Updated some installation steps after discussing it with Arc team well, link will not work untill it is published.